lib/modules/platform/github/index.ts

```diff
@@ -669,21 +653,12 @@ export async function getPr(prNo: number): Promise<Pr | null> {
     { prNo },
     'PR not found in open or closed PRs list - trying to fetch it directly'
   );
-  const pr = (
+  const ghRestPr = (
     await githubApi.getJson<GhRestPr>(
       `repos/${config.parentRepo || config.repository}/pulls/${prNo}`
     )
   ).body;
-  if (!pr) {
-    return null;
-  }
-  // Harmonise PR values
-  pr.displayNumber = `Pull Request #${pr.number}`;
-  if (pr.state === PrState.Open) {
```

Comment:
Seems like we didn't convert `merged_at && status == 'closed' => status: 'merged'` here